### ISSUE TITLE
kubectl process exit status lost with kubecolor

### ISSUE THREAD

**Issue by thomas-riccardi** [dty1er/kubecolor#41]
kubecolor ignores the exit status of the `kubectl` process:
```
$ /usr/bin/kubectl get pod foo; echo $?
Error from server (NotFound): pods "foo" not found
1
$ kubecolor get pod foo; echo $?
Error from server (NotFound): pods "foo" not found
0
```

(use-case: in a script (so the exit status is necessary), where a human still sees the outputs (so the color is much useful).)

**Comment by dty1er** [comment]
Nice catch. This should be fixed..

**Comment by thomas-riccardi** [comment]
Forwarding the errors from `cmd.Wait()` hopefully may be enough?

https://github.com/dty1er/kubecolor/blob/b83ce9d89601e9d8b3d37544f864a0d56d2faca9/command/runner.go#L96

**Comment by dty1er** [comment]
Currently this is fixed at the latest kubecolor (v0.0.11).
Thank you for opening this issue and sorry for my late work.

**Comment by thomas-riccardi** [comment]
Thanks @dty1er! I confirm 0.0.11 fixes the issue.

No worries for the time to fix: you owe nothing to the internet, thanks for your great work!
